src/main/java/uk/gov/hmcts/cmc/claimstore/models/DefendantLinkStatus.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.cmc.claimstore.models;
+
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.util.Objects;
+
+import static uk.gov.hmcts.cmc.claimstore.utils.ToStringStyle.ourStyle;
+
+public class DefendantLinkStatus {
+
+    private final boolean linked;
+
+    public DefendantLinkStatus(final boolean linked) {
+        this.linked = linked;
+    }
+
+    public boolean isLinked() {
+        return linked;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (this == other) {
+            return true;
+        }
+        if (other == null || getClass() != other.getClass()) {
+            return false;
+        }
+        DefendantLinkStatus status1 = (DefendantLinkStatus) other;
```

Comment:
`status1` -> `otherStatus`.

Reply:
odd

Reply:
but it's generated..

Reply:
`equals` code usually has to be manually adjusted after generating anyway because IDEA uses `null` comparisons and `instance.equals` instead of calling `Objects.equals`, so we may just as well rename I think.